bin/configs/avro-schema.yaml

```diff
@@ -1,4 +1,4 @@
 generatorName: avro-schema
 outputDir: samples/openapi3/schema/petstore/avro-schema
-inputSpec: modules/openapi-generator/src/test/resources/3_0/petstore.yaml
+inputSpec: modules/openapi-generator/src/test/resources/3_0/petstore-with-array-items-ref-to-enum.yaml
```

Comment:
Good idea to have a dedicated test files for avro-schema generator.

Please rename the file as `modules/openapi-generator/src/test/resources/3_0/avro-schema/petstore.yaml`

Reply:
Agreed and done